Snakefile

```diff
@@ -0,0 +1,69 @@
+
+DATASETS, = glob_wildcards("dataset_configs/{dataset}.yml")
+SUPERVISED, = glob_wildcards("model_configs/supervised/{supervised}.yml")
+UNSUPERVISED, = glob_wildcards("model_configs/unsupervised/{unsupervised}.yml")
+
+wildcard_constraints:
+    # Random seeds should be numbers
+    seed="\d+"
+
+rule all:
+    input:
+        # Pickled input dataframe
+        "data/subset_compendium.pkl",
+        # all_label_comparisons outputs 
+        expand("results/all_labels.{supervised}.{dataset}.{seed}.tsv", 
+               supervised=SUPERVISED,
+               dataset=DATASETS,
+               seed=range(0,5)
+               ),
+        # all_label_comparison_unsupervised
+        expand("results/all_labels.{unsupervised}.{supervised}.{dataset}.{seed}.tsv",
+               unsupervised=UNSUPERVISED,
+               supervised=SUPERVISED,
+               dataset=DATASETS,
+               seed=range(0,5)
+               ),
+
+rule pickle_compendium:
```

Comment:
Is loading a pickle faster than loading from a .tsv file? Or is there another reason you're doing this?

Reply:
Right, when I tested it loading the tsv to a dataframe takes ~3 minutes and loading the pickled dataframe takes ~2 seconds